lib/3scale/backend/transactor/status.rb

```diff
@@ -8,17 +8,21 @@ class Status
         # We only use 'redirect_uri' if a request sent such a param. See #397.
         REDIRECT_URI_FIELD = 'redirect_url'.freeze
         private_constant :REDIRECT_URI_FIELD
+        # Maximum number of keys to list when using the list_app_keys extension
```

Comment:
To avoid confusion, I think we should also clarify that system only allows to create 5.

Reply:
Yes, it could change anytime. We can start the comment with "At the time of writing this... " :writing_hand: 
But I think it's important to clarify.